### PR TITLE
Thread safe BreakpointStore

### PR DESCRIPTION
**Pull request by sgmunn**
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/949284

This removes all the locks around the breakpoint store now that it is thread safe and bumps to pick up the changes.

### CODE REVIEW

File: main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/BreakpointPad.cs [hunk 1]
```diff
@@ -421,33 +420,33 @@ public void UpdateDisplay ()
...
+					} else if (be is Catchpoint cp) {
+						name = cp.ExceptionName;
+					} else {
+						name = "";
 					}
+
+					store.AppendValues (GetIconId (be), be.Enabled, name, be, condition, traceExp, hitCount, traceVal);
 				}
+
+				store.AppendValues (GetIconId (be), be.Enabled, name, be, bp != null ? bp.ConditionExpression : null, traceExp, hitCount, traceVal);
```
Comment (mrward) [inline main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/BreakpointPad.cs]: This line seems to be causing a build failure. The original code does not seem to have this line.

```
MonoDevelop.Debugger/BreakpointPad.cs(449,36): error CS0103: The name 'be' does not exist in the current context [/Users/vsts/agent/2.155.1/work/1/s/monodevelop/main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger.csproj]
```

Comment (sgmunn) [reply main/src/addins/MonoDevelop.Debugger/MonoDevelop.Debugger/BreakpointPad.cs]: thanks @mrward I must have missed this on the rebase

**Comment by sgmunn** [comment]
debuggerlibs merged, this bumps to the tip of master so is safe to merge

**Comment by sgmunn** [comment]
`TestCreateBuildHttpTrigger` timed out again, unrelated.

**Pull request merged**